bedrock/firefox/templates/firefox/lockwise/lockwise.html

```diff
@@ -0,0 +1,105 @@
+{% add_lang_files "firefox/products/lockwise" %}
+
+{% from "macros.html" import google_play_button with context %}
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_favicon %}{{ static('img/firefox/lockwise/favicon.ico') }}{% endblock %}
+
+{% block page_title %}{{_('Firefox Lockwise — password manager —  take your passwords everywhere')}}{% endblock %}
+
+{% block page_desc %}{{ _('Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser. Features 256-bit encryption and Face/Touch ID.') }}{% endblock %}
```

Comment:
You can also declare a Lockwise specific social sharing image for this page like this:

```
{% block page_image %}{{ static('/protocol/img/logos/firefox/lockwise/og.png') }}{% endblock %}
```